### DISCUSSION TITLE
milvus根据id查询feature性能问题

### DISCUSSION THREAD

**Discussion by buptzouy** [milvus-io/milvus#19582] [Q&A and General discussion]
如果数据量超过5000w，频繁根据ids查询features，milvus能撑住吗？谢谢

**Comment by yhmo** [comment]
这部分我们还没来得及优化，单次调用耗时上百毫秒都是有可能的，我估计目前不太撑得住。

**Reply by buptzouy** [reply]
> 这部分我们还没来得及优化，单次调用耗时上百毫秒都是有可能的，我估计目前不太撑得住。

好的，谢谢
